spec/features/audit/allocation/allocate_spec.rb

```diff
@@ -126,36 +126,83 @@
     let!(:books) do
       create_list(
         :content_item,
-        20,
+        60,
         primary_publishing_organisation: novelists,
       )
     end
 
     let!(:paintings) do
       create_list(
         :content_item,
-        20,
+        60,
         primary_publishing_organisation: painters,
       )
     end
 
-    scenario "I can assign 26 content items to myself from both organisations" do
+    scenario "I can assign 101 content items to myself from both organisations" do
       visit audits_allocations_path
 
-      expect(page).to have_content("20 items")
+      expect(page).to have_content("60 items")
 
       page.select "Painters", from: "organisations[]"
       click_on "Apply filters"
 
       expect(page).to have_select("organisations[]", selected: %w[Novelists Painters])
-      expect(page).to have_content("40 items")
+      expect(page).to have_content("120 items")
 
       select "Me", from: "allocate_to"
-      fill_in "batch_size", with: "26"
+      fill_in "batch_size", with: "101"
       click_on "Assign"
 
-      expect(page).to have_content("26 items assigned to Jane Austen")
-      expect(page).to have_content("14 items")
+      expect(page).to have_content("101 items assigned to Jane Austen")
+      expect(page).to have_content("19 items")
+    end
+  end
+
+  context "There three pages of content items" do
+    let!(:content_items) do
+      create_list(
+        :content_item,
+        210,
+        primary_publishing_organisation: novelists,
+      )
+    end
+
+    scenario "I can assign over one page of items starting on the second page" do
+      visit audits_allocations_path
+
+      expect(page).to have_content('210 items')
+
+      # Collect the titles that span each page, 100 per page
+      page_1_titles = content_item_titles
+      go_to_page 3
+      page_3_titles = content_item_titles
+      go_to_page 2
+
+      select 'Me', from: 'allocate_to'
```

Comment:
Isn't this a good opportunity to move into the new style of testing? We are adding new tests with the old way of doing things.

Reply:
The page objects are not available to work with in master yet so I chose not to step on the shoes of other branches.